pipeline/06-upload.R

```diff
@@ -16,7 +16,6 @@ metadata <- read_parquet(paths$output$metadata$local)
 cv_enable <- metadata$cv_enable
 shap_enable <- metadata$shap_enable
 run_id <- metadata$run_id
```

Comment:
**[Thought, non-blocking]** Unrelated to the changes in this PR, but I just noticed: Now that we're loading these parameters in all scripts via sourcing the `R/setup.R` script, I don't think we need to load these params from the metadata file, right? No need to address this here, we can also do it in a follow-up!

Reply:
`run_id` isn't actually created until `finalize`, so we do need to load at least that from the metadata. There other two will load directly from setup. I'll fix in an upcoming PR.